src/lib/FreeC/Backend/Agda/Pretty.hs

```diff
@@ -0,0 +1,44 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+-- | This module contains 'Pretty' instances for the Agda AST.
+--
+--   The Agda has its own 'Agda.Pretty' class in 'Agda.Utils.Pretty'. Therefore
```

Comment:
`Agda.Utils.Pretty` is a module. Thus, double quotes are needed in Haddock to link to it.

```suggestion
--   The Agda library has its own 'Agda.Pretty' class in "Agda.Utils.Pretty". Therefore
```